tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1546,6 +1546,31 @@ std::ostream& operator<<(std::ostream& os, const TestParamBase& p) {
   return os;
 }
 
+template<typename T>const char *format();
+
+template<> const char *format<int>()      { return "%d"; }
+template<> const char *format<float>()    { return "%.1f"; }
+
+template<typename T>
+void out_vector(const std::vector<T> &vector, const char *pComment, const string &name, const string &type="") {
+
```

Comment:
I think it is better to move if (VLOG_IS_ON(2)) from the call sites of this routine to here.
What do you think?

Reply:
I think you are right and I will make the appropriate changes